src/components/Items/Scorecard/Scorecard.vue

```diff
@@ -0,0 +1,252 @@
+        <div class="text-center text-base md:text-xl lg:text-2xl font-extrabold">
+          {{ plioTitle }}
+        </div>
+
+        <!-- scorecard greeting -->
+        <div class="text-center text-sm md:text-lg lg:text-xl font-extrabold">
+          {{ greeting }}
+        </div>
+
+        <!-- canvas element for drawing the confetti -->
+        <canvas id="confetticanvas" class="fixed z-50"></canvas>
+
+        <!-- circular progress bar -->
+        <CircularProgress
+          class="relative mx-auto"
+          :radius="circularProgressRadius"
+          :progress="localProgressPercentage"
+          :stroke="circularProgressStroke"
+          :progressNumberIndicator="progressNumberIndicator"
+          :key="reRenderKey"
+        >
+        </CircularProgress>
+
+        <!-- metric boxes -->
+        <div class="flex flex-row justify-between gap-4">
+          <div
+            v-for="metric in metrics"
+            class="w-full rounded-2xl bg-pastel-yellow grid grid-rows-2 lg:grid-rows-3 border-2"
+            :key="metric"
+          >
+            <div class="lg:row-span-2 w-full h-full flex flex-row justify-center gap-2">
+              <inline-svg
+                :src="getIconSource(metric.icon.source)"
+                class="h-4 md:h-6 lg:h-10 w-4 md:w-6 lg:w-10 place-self-center"
+                :class="metric.icon.color"
+              ></inline-svg>
+              <p class="text-base md:text-2xl lg:text-4xl font-bold my-auto">
+                {{ metric.value }}
+              </p>
+            </div>
+            <div class="text-center text-sm md:text-base lg:text-xl font-medium">
+              {{ metric.description }}
+            </div>
+          </div>
+        </div>
+
+        <!-- action buttons -->
+        <div class="place-self-center mt-3">
+          <icon-button
+            :titleConfig="watchAgainButtonTitleConfig"
+            :buttonClass="watchAgainButtonClass"
+            @click="watchAgain"
+            data-test="watchAgainButton"
+          ></icon-button>
+        </div>
+      </div>
+    </div>
+  </div>
+</template>
+
+<script>
+import CircularProgress from "@/components/UI/Progress/CircularProgress.vue";
+import Utilties from "@/services/Functional/Utilities.js";
+import IconButton from "@/components/UI/Buttons/IconButton.vue";
+
+const confetti = require("canvas-confetti");
+const confettiCanvas = document.getElementById("confetticanvas");
+const confettiHandler = confetti.create(confettiCanvas, { resize: true });
+
+export default {
+  name: "Scorecard",
+  components: {
+    CircularProgress,
+    IconButton,
+  },
+  props: {
+    metrics: {
+      // details of all the metrics to show
+      // format of one metric --
+      // {
+      //   name: "METRIC_NAME",
+      //   icon: {
+      //     source: "SVG_NAME.svg",
+      //     color: "TAILWIND_CLASSES",
+      //   },
+      //   value: VALUE_TO_SHOW,
+      //   description: DESCRIPTION_TO_SHOW,
```

Comment:
what is the difference between name and description? I don't see name being used anywhere?